src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -189,6 +193,18 @@ function PaymentMethodList(props) {
      */
     const renderListEmptyComponent = useCallback(() => <Text style={[styles.popoverMenuItem]}>{translate('paymentMethodList.addFirstPaymentMethod')}</Text>, [translate]);
 
+    const renderListFooterComponent = useCallback(
+        () => (
+            <MenuItem
+                onPress={props.onPress}
+                title={translate('walletPage.addBankAccount')}
+                icon={Expensicons.Plus}
+                wrapperStyle={[styles.cardMenuItem]}
+            />
+        ),
+        [props.onPress, translate],
```

Comment:
Components that use this `PaymentMethodList`, are sending not stable function, so this component will be recreated all the time, not sure if the `useCallback` will work now

Reply:
I think `useCallback` could be useful here. Since, the deps. array is pretty small. We can test it via a simple log 🙂